schema-shared.graphql

```diff
@@ -75,6 +75,11 @@ type CollectionPartnership {
   blurb: Markdown!
 }
 
+type Label {
+  labelExternalId: ID!
+  labelName: String!
```

Comment:
I think we should name these without the `label` prefix. That way we can keep it consistent with the rest of property names of the other types!